.pre-commit-config.yaml

```diff
@@ -22,20 +22,21 @@ repos:
         language: system
         types: [file, rust]
         entry: cargo fmt -- --check
+        pass_filenames: false
```

Comment:
Why is this needed?

Reply:
Cargo fmt doesn't need passing filenames and `find . -name "*.rs"` includes two somehow unformatted, but also empty, main.rs files in tests.

Reply:
Got it!